clarifying device support for deprecated and unsupported (#209)
ROCm/rocm-install-on-linux: docs/reference/system-requirements.rst

      "AMD Radeon RX 7900 GRE", "RDNA3", "gfx1100", "✅"
      "AMD Radeon VII", "GCN5.1", "gfx906", "⚠️"

> ✅: **Supported** - Official software distributions of the current ROCm release fully enables this hardware.

Reviewer: ```suggestion
✅: **Supported** - Official software distributions of the current ROCm release fully support this hardware.
```
"enables" sounds weird to me. "supports" sounds more natural to me in this context, but not a blocker.

Also, we should use the appropriate verb tense here. THanks

> ✅: **Supported** - AMD enables these GPUs in our software distributions for the corresponding

Reviewer: ```suggestion
✅: **Supported** - Official software distributions of the current ROCm release fully support this hardware. All features and optimizations work as expected.
```

ROCm product.

> ⚠️: **Deprecated** - All existing GPU functionality will be maintained, and no new functionality will be added.

Reviewer: ```suggestion
⚠️: **Deprecated** - The current ROCm release has limited support for this hardware. Existing features and capabilities are maintained, but no new features or optimizations will be added. A future ROCm release will remove support. Consider upgrading to fully supported hardware.
```

> ⚠️: **Deprecated** - Support for deprecated devices will be removed in a future release. While all existing functionalities on these GPUs will be maintained, no new functionalities may be added.

Reviewer 2: Can a device be deprecated?

Reviewer 2: ```suggestion
⚠️: **Deprecated** - All existing GPU functionality will be maintained, and no new functionality will be added.
```

> ❌: **Unsupported** - HIP runtime might continue to perform as expected, but prebuilt ROCm libraries published by AMD are no longer supported and will result in runtime errors.

Reviewer: ```suggestion
❌: **Unsupported** - The current ROCm release does not support this hardware. The HIP runtime can continue to work, but prebuilt ROCm libraries are not officially supported and will cause runtime errors. Consider upgrading to fully supported hardware.
```

> ❌: **Unsupported** - HIP runtime may continue to work but prebuilt ROCm libraries published by AMD no longer support your product and will cause run-time errors.

Reviewer 2: ```suggestion
❌: **Unsupported** - HIP runtime might continue to perform as expected, but prebuilt ROCm libraries published by AMD are no longer supported and will result in runtime errors.
```
